_pages/how-we-work/equipment.md

```diff
@@ -100,17 +100,9 @@ For issues with any GSA-provided hardware (such as laptops or phones), [contact
 
 ## Lost or stolen equipment
 
-If TTS-issued equipment is lost or stolen, follow these mandatory steps. Employees must follow the GSA procedures for lost or stolen equipment or missing PIV cards, and alert TTS in [#equipment](https://gsa-tts.slack.com/messages/equipment):
-
-1. Inform [#equipment](https://gsa-tts.slack.com/messages/equipment) within one business day of learning the equipment is stolen or lost.
-2. Call 911 or a local police station when you suspect theft has occurred.
-3. Get and keep a copy of the police report in case you have to provide it to the Federal Protective Service (FPS) or your regional Information Systems Security Officer (ISSO). Provide as much information as possible.
-4. [Contact the GSA IT Service Desk.]({{ "/gsa-internal-tools/#it-service-desk" | relative_url }}) An agent will create a ticket and initiate the process of wiping data off any stolen mobile device or laptop.
-5. Call your FPS using contact information provided in the link.
-6. Contact your regional ISSO using contact information provided in the link.
-7. If your equipment is returned to you, it may have been tampered with and should not be used. Return the equipment to GSA IT so they can inspect it, and if necessary, re-image it.
-
-TTS will replace stolen or lost equipment once a copy of the police report is sent to Team Ops and the employee confirms they contacted FPS. If you lose your PIV card, you will also need to fill out this [lost access card form](https://drive.google.com/file/d/0B0SnS9Ferbdnc3p6cmdOT05mUGNwdlNLMncxME5VZFlOOFhz/view) and email it to [hspd12.security@gsa.gov](mailto:hspd12.security@gsa.gov)
+- [PIV cards](https://insite.gsa.gov/topics/safety-and-security/access-cards-and-real-id/access-cards-new-lost-or-stolen-cards)
+- [All other equipment (phones, laptops, etc.)](https://insite.gsa.gov/topics/information-technology/assistance-and-help-desks/report-stolen-equipment)
```

Comment:
Here's my concern: in the case of lost or stolen equipment, the person with the lost or stolen equipment may have also just lost their 2FA access point, so it seems like we should give them the key things they need to do _right now_ without forcing them to figure that out first.